### DISCUSSION TITLE
Allow defining the dashboard variables values for the links build by the alerting notifier

### DISCUSSION THREAD

**Discussion by Vimao** [grafana/grafana#44580] [Alerting]
Currently the NG Alerting notifier builds URL without any additional parameters.

It would be great if we could specify in the alert what values should the variables on the dashboard be, without having to manually copy the pre-created dashboard link into a field in the notification. 

I think there should be a field which would allow you more control over the URL the notifier is building which would accept the parameters and create the url for the dashboard or panel.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
